### PR TITLE
[jasper] freeglut is not a dependency in macOS

### PR DESCRIPTION
**Pull request by david-antiteum**
**Describe the pull request**

Removing freeglut from macOS list of dependencies.

- What does your PR fix? Fixes issue

Broken Qt 5 in macOS after commit https://github.com/microsoft/vcpkg/commit/f8165f72709cce797058164b389a105ca4c412a9#diff-5df6afe35378f98a8c68a59e1faed145

### CODE REVIEW

File: ports/jasper/CONTROL [hunk 1]
```diff
@@ -2,4 +2,4 @@ Source: jasper
 Version: 2.0.16-2
```
Comment (JackBoosY) [inline ports/jasper/CONTROL]: Can you update version info? see [documentation](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md#update-the-version-field-in-the-control-file-of-any-modified-ports).

**Review by JackBoosY — CHANGES_REQUESTED** [review]
Can you remove `freeimage:x64-osx=fail` in _VCPKG_PATH/scripts/ci.baseline.txt_ ?

**Comment by Neumann-A** [comment]
duplicate #9680. This one should probably be closed because the source repo has been deleted?

**Comment by david-antiteum** [comment]
@Neumann-A Closing. Yes, #9680 is the good one.

**Pull request closed without merging**